core/src/test/scala/unit/kafka/server/ControllerMutationQuotaTest.scala

```diff
@@ -364,7 +385,7 @@ class ControllerMutationQuotaTest extends BaseRequestTest {
   }
 
   private def waitUserQuota(user: String, expectedQuota: Double): Unit = {
-    val quotaManager = servers.head.quotaManagers.controllerMutation
+    val quotaManager = brokers.head.quotaManagers.controllerMutation
```

Comment:
In KRaft mode should we check the QuotaManager on both `brokers.head` and `controllerServers.head`?

Reply:
Fixed